android/src/main/java/com/pushwithgcm/PushWithGCMListenerService.java

```diff
@@ -0,0 +1,6 @@
+package com.pushwithgcm;
+
+import com.google.android.gms.gcm.GcmListenerService;
+
+public class PushWithGCMListenerService extends GcmListenerService {
```

Comment:
We must have a GCMListenerService for push notifications to work.
Since we use GCM Messages of type "notification", no extra handling is needed.